ccd-definition/AuthorisationCaseField/CareSupervision/magistrate.json

```diff
@@ -1273,6 +1273,48 @@
     "UserRole": "caseworker-publiclaw-magistrate",
     "CRUD": "R"
   },
+  {
```

Comment:
does magistrate need R perms for these if they aren't displayed in tab and they dont have access to the event?

Reply:
magistrate role doesn't have access to the event, so removed fields from this file